shared/common-adapters/avatar.desktop.js

```diff
@@ -19,23 +20,11 @@ export default class Avatar extends Component {
     this.state = {avatarLoaded: false}
   }
 
-  _createUrl (): ?string {
-    if (__SCREENSHOT__) {
-      return noAvatar
-    } else if (this.props.url) {
-      return this.props.url
-    } else if (this.props.username) {
-      return `https://keybase.io/${this.props.username}/picture`
-    }
-
-    return null
-  }
-
   render () {
     const {size} = this.props
     const width = size
     const height = size
-    const url = this._createUrl()
+    const url = __SCREENSHOT__ ? noAvatar : shared.createAvatarUrl(this.props)
```

Comment:
Sure thing. I'll pull it back into mobile. My concern was ReferenceErrors, but I'll just ensure that `__SCREENSHOT__` is set to false on native.